source/MaterialXRuntime/Private/PrvStage.cpp

```diff
@@ -27,12 +29,6 @@ PrvObjectHandle PrvStage::createNew(const RtToken& name)
     return std::make_shared<PrvStage>(name);
 }
 
-void PrvStage::initialize()
```

Comment:
Without this. A new stage needs to be created, and all dependent references moved over. Unsure, but if the dependents ref counts go to 0, then think they can disappear before they can be re-referenced?

Reply:
I imagine and application, like our node editor, will keep a separate stage where it manages all loaded libraries (keeping a reference to it so all libraries are kept alive). And when a new working stage is needed you create it and add the single reference to this library stage.

So there should be no need to track many different referenced libraries for each working stage, and move them over if you want to create a new working stage.